server/src/main/resources/org/elasticsearch/common/reference-docs-links.txt

```diff
@@ -1,48 +1,48 @@
+BOOTSTRAP_CHECK_MAXIMUM_MAP_COUNT                               deploy-manage/deploy/self-managed/bootstrap-checks-max-map-count
+BOOTSTRAP_CHECK_CLIENT_JVM                                      deploy-manage/deploy/self-managed/bootstrap-checks-client-jvm
+BOOTSTRAP_CHECK_USE_SERIAL_COLLECTOR                            deploy-manage/deploy/self-managed/bootstrap-checks-serial-collector
+BOOTSTRAP_CHECK_SYSTEM_CALL_FILTER                              deploy-manage/deploy/self-managed/bootstrap-checks-syscall-filter
+BOOTSTRAP_CHECK_ONERROR_AND_ONOUTOFMEMORYERROR                  deploy-manage/deploy/self-managed/bootstrap-checks-onerror
+BOOTSTRAP_CHECK_EARLY_ACCESS                                    deploy-manage/deploy/self-managed/bootstrap-checks-early-access
+BOOTSTRAP_CHECK_ALL_PERMISSION                                  deploy-manage/deploy/self-managed/bootstrap-checks-all-permission
+BOOTSTRAP_CHECK_DISCOVERY_CONFIGURATION                         deploy-manage/deploy/self-managed/bootstrap-checks-discovery-configuration
+BOOTSTRAP_CHECKS                                                deploy-manage/deploy/self-managed/bootstrap-checks
+BOOTSTRAP_CHECK_ENCRYPT_SENSITIVE_DATA                          deploy-manage/deploy/self-managed/bootstrap-checks-xpack#bootstrap-checks-xpack-encrypt-sensitive-data
+BOOTSTRAP_CHECK_PKI_REALM                                       deploy-manage/deploy/self-managed/bootstrap-checks-xpack#bootstrap-checks-xpack-pki-realm
+BOOTSTRAP_CHECK_ROLE_MAPPINGS                                   deploy-manage/deploy/self-managed/bootstrap-checks-xpack#bootstrap-checks-xpack-role-mappings
+BOOTSTRAP_CHECK_TLS                                             deploy-manage/deploy/self-managed/bootstrap-checks-xpack#bootstrap-checks-tls
+BOOTSTRAP_CHECK_TOKEN_SSL                                       deploy-manage/deploy/self-managed/bootstrap-checks-xpack#bootstrap-checks-xpack-token-ssl
+CONTACT_SUPPORT                                                 troubleshoot/elasticsearch/elasticsearch#troubleshooting-contact-support
+UNASSIGNED_SHARDS                                               troubleshoot/elasticsearch/red-yellow-cluster-status
+EXECUTABLE_JNA_TMPDIR                                           deploy-manage/deploy/self-managed/executable-jna-tmpdir
+NETWORK_THREADING_MODEL                                         reference/elasticsearch/configuration-reference/networking-settings#modules-network-threading-model
+ALLOCATION_EXPLAIN_API                                          api/doc/elasticsearch/v9/operation/operation-cluster-allocation-explain
+NETWORK_BINDING_AND_PUBLISHING                                  reference/elasticsearch/configuration-reference/networking-settings#modules-network-binding-publishing
+SNAPSHOT_REPOSITORY_ANALYSIS                                    api/doc/elasticsearch/v9/operation/operation-snapshot-repository-analyze
```

Comment:
> SNAPSHOT_REPOSITORY_ANALYSIS 
This is another link to the API docs that don't support the same query parameter or URL structure as the other docs. i.e. it would need to use a path of https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-snapshot-repository-analyze or https://www.elastic.co/docs/api/doc/elasticsearch/v9/operation/operation-snapshot-repository-analyze